src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.WorkspaceEventListener.cs

```diff
@@ -18,14 +18,6 @@ public void StartListening(Workspace workspace, object serviceOpt)
                 var service = (RemoteHostClientService)workspace.Services.GetService<IRemoteHostClientService>();
                 service.Enable();
             }
-
-            public void Stop(Workspace workspace)
```

Comment:
not ever called?

Reply:
Correct.

---
In reply to: [360169298](https://github.com/dotnet/roslyn/pull/40395#discussion_r360169298) [](ancestors = 360169298)